src/frontend/src/lib/components/tokens/TokenLogo.svelte

```diff
@@ -25,7 +27,13 @@
 		{color}
 		{ring}
 	/>
-	{#if showNetworkIcon}
+	{#if nonNullish(tokenCount) && tokenCount > 0}
+		<span
```

Comment:
a11y?

Reply:
sorry, forgot! Have a look, i put 

`"token_group_number": "Total number of tokens in the group for native token $token"`

Reply:
or shall i add the number there too?

`"token_group_number": "There are $count tokens in the group for native token $token"`

Reply:
I guess the current implementation - `"token_group_number": "Total number of tokens in the group for native token $token"` - is fine because the number itself contained in the span is also readable. Thx.